alert-database/src/main/java/com/synopsys/integration/alert/database/api/ClientCertificateAndKeyAccessor.java

```diff
@@ -0,0 +1,113 @@
+package com.synopsys.integration.alert.database.api;
+
+import static com.synopsys.integration.alert.common.rest.AlertRestConstants.DEFAULT_CLIENT_CERTIFICATE_ALIAS;
+import static com.synopsys.integration.alert.common.rest.AlertRestConstants.DEFAULT_CONFIGURATION_NAME;
+
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.api.common.model.exception.AlertConfigurationException;
+import com.synopsys.integration.alert.common.persistence.accessor.UniqueConfigurationAccessor;
+import com.synopsys.integration.alert.common.persistence.model.ClientCertificateAndKeyModel;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.security.EncryptionUtility;
+import com.synopsys.integration.alert.common.util.DateUtils;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateEntity;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateKeyEntity;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateKeyRepository;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateRepository;
+
+import jakarta.transaction.Transactional;
+
+@Component
+@Transactional
+public class ClientCertificateAndKeyAccessor implements UniqueConfigurationAccessor<ClientCertificateAndKeyModel> {
+    private final EncryptionUtility encryptionUtility;
+    private final ClientCertificateKeyRepository keyRepository;
+    private final ClientCertificateRepository certificateRepository;
+
+    public ClientCertificateAndKeyAccessor(EncryptionUtility encryptionUtility, ClientCertificateKeyRepository keyRepository, ClientCertificateRepository certificateRepository) {
+        this.encryptionUtility = encryptionUtility;
+        this.keyRepository = keyRepository;
+        this.certificateRepository = certificateRepository;
+    }
+
+    @Override
+    public Optional<ClientCertificateAndKeyModel> getConfiguration() {
+        Optional<ClientCertificateKeyEntity> optionalKeyEntity =
+            keyRepository.findByName(DEFAULT_CONFIGURATION_NAME);
+        Optional<ClientCertificateEntity> optionalCertificateEntity =
+            certificateRepository.findByAlias(DEFAULT_CLIENT_CERTIFICATE_ALIAS);
```

Comment:
Minor formatting point, but do these need to be split on separate lines, or does it go over line character limit?